src/utils/constants/constants.jsx

```diff
@@ -0,0 +1,24 @@
+import markIcon from "src/assets/images/mark.png";
+import penIcon from "src/assets/images/pen.png";
+import binIcon from "src/assets/images/bin.png";
+import logo from "src/assets/images/logo.png";
+import navLogo from "src/assets/images/nav-logo.png";
+import searchIcon from "src/assets/images/search.png";
+import addIcon from "src/assets/images/add.png";
+
+export const MARK_TEXT = "Mark task as done";
+export const EDIT_TEXT = "Edit task";
+export const DELETE_TEXT = "Delete task";
+export const LOGO_TEXT = "Logo";
+export const SEARCH_TEXT = "Search";
+export const NAV_LOGO_TEXT = "Logo";
+
+export const [
+  MARK_ICON,
```

Comment:
can we add common prefix to these const ? [reference](https://github.com/azo-wtag/todo-app-react/pull/4#discussion_r1172083576)
```suggestion
  ICON_MARK,
```